cli/cli-demo.ts

```
@@ -0,0 +1,135 @@
+
+  // 2. Literature search
+  console.log("2) Searching PubMed for sickle-cell literature (top 3 IDs)...")
+  const pubmedIds = await session.searchLiterature("sickle cell vaso-occlusive event biomarkers", 3)
+  console.log("PubMed IDs:", pubmedIds.join(", "), "\n")
+
+  // 3. Medical data analysis (very small CSV example)
+  console.log("3) Performing basic data analysis on sample CSV...")
+  const csvData = "value1,value2\n1,2\n3,4"
+  const analysisRes = await session.analyzeMedicalData({
+    analysisType: "data_analysis" as any,
+    data: csvData,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)